PortalFetch/crawler_data_process.py

```diff
@@ -42,7 +46,7 @@ def __save_contents(self, filename, firstline, urlist):
             writer = csv.writer(f)
             writer.writerow([firstline])
 
-            for i in range(8, len(urlist)-1):
+            for i in range(7, len(urlist)-1):
```

Comment:
Should we define some macro variable so we know what ```7``` means in the future?
Otherwise it LGTM, we need to change from saving to ```.csv``` to saving to ```.json``` in the next PR tho